### ISSUE TITLE
CI failing for release job

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#2790]
Release job failing with following error. 

https://ci.centos.org/job/minishift-release/109/console

```
06:24:31   Scenario: OpenShift is ready after startup # features/basic.feature:32
06:24:31     And with up to "20" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200" # features/basic.feature:39
06:24:31       Error: status code of response from https://192.168.42.130:8443/console is not equal to expected string. Expected: 200, Actual: 502
```

**Comment by praveenkumar** [comment]
@LalatenduMohanty I think we should update the description of this issue since it is not only related to release but also nightly, master and pr side.

**Comment by praveenkumar** [comment]
resolved by #2793
